skopt/searchcv.py

```diff
@@ -456,13 +456,7 @@ def _store(key_name, array, weights=None, splits=False, rank=False):
                 results["rank_%s" % key_name] = np.asarray(
                     rankdata(-array_means, method='min'), dtype=np.int32)
 
-        # Computed the (weighted) mean and std for test scores alone
-        # NOTE test_sample counts (weights) remain the same for all candidates
-        test_sample_counts = np.array(test_sample_counts[:n_splits],
-                                      dtype=np.int)
```

Comment:
Why is this being removed as well?

Reply:
Good that you should mention it. I found some extra code to purge. :smile: 

Might make sense to _replace_ it all instead of purging, but I'm not the one who understands it enough for that.